### DISCUSSION TITLE
File attachments for Waline

### DISCUSSION THREAD

**Discussion by hiensarahly** [walinejs/waline#847] [Ideas]
Hi! I’ve been using Waline for a while now and my audience seem to like it as well. There is one guy who suggests an attachment feature of files like word .doc or pdf. So I’d like to ask if this would be a possible feature to develop with the current comment system? I’d like to share this idea for consideration for the improvement of Waline.

Thanks!

**Comment by Mister-Hope** [comment]
We can surely support this feature, but it should not spoil current code heavily and should not be a burden to users who don't need it.

Just to be curious, how can we gracefully store files in database, or in other word, where do you want to store them?

Also IMO, I disagree this is a common need, as in most situations people would only want uploading images in comment. So I strongly recommand you to develop a page for your users to upload files if needed.

**Reply by hiensarahly** [reply]
Makes sense. I’ll try another solution though. Thank u very much for recommending.😊
